contributed_definitions/NXellipsometry.nxdl.xml

```diff
@@ -188,7 +209,16 @@ A rework of the draft version(06/2022) of a NeXus application definition for ell
                      If focussing probes (lenses) were used, please state if the data
                      were corrected for the window effects.
                 </doc>
-                <!--This as well can be stated in window/aperture-->
+                <field name="type">
+                    <enumeration>
+                        <item value="objective"/>
+                        <item value="lens"/>
+                        <item value="glass fiber"/>
+                        <item value="none"/>
+                        <item value="other"/>
+                    </enumeration>
+                </field>
+                <group name="device_information" type="NXfabrication" optional="true"/>
```

Comment:
Typically when you have a group in an appdef that is optional but none of its childs are required, one should remove the group because you can always add optional groups. The situation becomes different if you say, okay fabrication is optional but if people use this they need to report e.g. vendor

Reply:
yea, its the latter case, so that allways the device information is saved exactly at these fields. This came originally from mpes, and I thaught it would be good. But maybe it just bloats unnecessary these definition. I added a note, that this may be reworked...